Makefile

```diff
@@ -94,6 +109,7 @@ undeploy: ## Undeploy controller from the K8s cluster specified in ~/.kube/confi
 
 OS := $(shell uname -s | tr '[:upper:]' '[:lower:]')
 ARCH := $(shell uname -m | sed -e 's/x86_64/amd64/' -e 's/aarch64/arm64/')
+ARCH2 := $(shell uname -m | sed -e 's/aarch64/arm64/')
```

Comment:
Why is this needed? This looks like it should output the same thing as the line above it. What am I missing?

Reply:
My mind must have been broken for a minute :)